src/frontend/context/OppgaverContext.tsx

```diff
@@ -59,6 +76,46 @@ const [OppgaverProvider, useOppgaver] = createUseContext(() => {
         );
     };
 
+    const fordelOppgave = (oppgave: IOppgave, saksbehandler: string): Promise<Ressurs<string>> => {
+        return axiosRequest<string, void>({
+            method: 'POST',
+            url: `/familie-ba-sak/api/oppgave/${oppgave.id}/fordel?saksbehandler=${saksbehandler}`,
+        })
+            .then((oppgaverRes: Ressurs<string>) => {
+                if (
+                    OppgavetypeFilter[oppgave.oppgavetype as keyof typeof OppgavetypeFilter] ===
+                    OppgavetypeFilter.JFR
+                ) {
+                    history.push(`/oppgaver/journalfør/${oppgave.id}`);
+                } else {
+                    opprettEllerHentFagsak({
```

Comment:
Er det riktig at vi alltid skal opprette fagsak her?

Reply:
Teoretisk sett, nei. For Godkjenne Vedtak/Behandle Underkjent Vedtak-oppgaver gir det ikke mening å opprette en ny fagsak. Men jeg tror aldri man vil havne i en situasjon hvor man plukker en slik oppgave og fagsaken ikke finnes fra før, for disse oppgavene opprettes ifm totrinnskontrollen på behandlingen. 

For BehandleSak-oppgaver tenker jeg at det gir mening å opprette fagsaken hvis den ikke finnes fra før. Ellers er det jo en drøss med andre oppgavetyper som kommer opp på enheten. De har vi jo ikke støtte for å utføre enda, så en mulig løsning der kan være å ikke ha noen plukk-knapp for andre oppgavetyper enn de vi håndterer nå.

Reply:
Ja, tror vi skal være litt forsiktig med å opprette fagsak her. Tenker vi bør kun legge inn støtte for typene som skal behandles hos oss.

Reply:
La inn fiks for dette :) c070f67